From 39f9fe4a47b1bb564c72944b624889d48dda2c3d Mon Sep 17 00:00:00 2001
From: Sabrina Dubroca <sdubroca@redhat.com>
Date: Fri, 9 Jun 2017 10:08:00 +0200
Subject: [PATCH] devlink: whitespace cleanup

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1459772
Upstream Status: iproute2.git commit 7c55d7700f43

commit 7c55d7700f4320186784590516993a73183bc2f2
Author: Stephen Hemminger <stephen@networkplumber.org>
Date:   Mon Aug 29 11:17:38 2016 -0700

    devlink: whitespace cleanup

    Break long lines

Signed-off-by: Sabrina Dubroca <sdubroca@redhat.com>
---
 devlink/devlink.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/devlink/devlink.c b/devlink/devlink.c
index ab55f8c73681..52fc7f49f51c 100644
--- a/devlink/devlink.c
+++ b/devlink/devlink.c
@@ -659,7 +659,8 @@ static int threshold_type_get(const char *typestr,
 	return 0;
 }
 
-static int eswitch_mode_get(const char *typestr, enum devlink_eswitch_mode *p_mode)
+static int eswitch_mode_get(const char *typestr,
+			    enum devlink_eswitch_mode *p_mode)
 {
 	if (strcmp(typestr, ESWITCH_MODE_LEGACY) == 0) {
 		*p_mode = DEVLINK_ESWITCH_MODE_LEGACY;
@@ -784,6 +785,7 @@ static int dl_argv_parse(struct dl *dl, uint32_t o_required,
 		} else if (dl_argv_match(dl, "mode") &&
 			   (o_all & DL_OPT_ESWITCH_MODE)) {
 			const char *typestr;
+
 			dl_arg_inc(dl);
 			err = dl_argv_str(dl, &typestr);
 			if (err)
@@ -846,7 +848,8 @@ static int dl_argv_parse(struct dl *dl, uint32_t o_required,
 		return -EINVAL;
 	}
 
-	if ((o_required & DL_OPT_ESWITCH_MODE) && !(o_found & DL_OPT_ESWITCH_MODE)) {
+	if ((o_required & DL_OPT_ESWITCH_MODE) &&
+	    !(o_found & DL_OPT_ESWITCH_MODE)) {
 		pr_err("E-Switch mode option expected.\n");
 		return -EINVAL;
 	}
-- 
2.13.1

